src/components/SyncPurchases.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios'
+import config from '../config'
+
+const SyncPurchases = {
+    job: false,
+    startJob: () => {
+        if (!SyncPurchases.job) {
+            setInterval(SyncPurchases.syncData, 20000);
+            SyncPurchases.job = true;
+        }
+    },
+    purchaseKey: () => {
+        return `pcs::${new Date().getTime()}`
+    },
+    storePurchase: (purchase) => {
+        localStorage.setItem(SyncPurchases.purchaseKey(), JSON.stringify(purchase));
+    },
+    syncData: async () => {
+        for (var key in localStorage) {
+            if (key.indexOf('pcs::') === 0) {
+                var purchase = JSON.parse(localStorage.getItem(key))
```

Comment:
Do we need to parse?

Reply:
Yes, the localStorage only supports storage strings, then I convert to string and then parse the content.

Reply:
https://github.com/izn/flipermercado-sheets/pull/9/files#diff-0294be1739eee5728f433d23be44fffcR16